### DISCUSSION TITLE
Is it possible to enable the privacy.resistFingerprinting features individually?

### DISCUSSION THREAD

**Discussion by vickygamer997** [Floorp-Projects/Floorp#929] [Q&A] [answered]
Because if it's not, I'll try to add this to the browser myself.

**Comment by CutterKnife** [comment]
What do you mean by "enable individually"?

**Comment by vickygamer997** [comment]
I mean to be able to turn on the user agent change and canvas blocker, without other changes.

**Reply by CutterKnife** [reply]
Hmm... not sure if this is what you want, but there is an anti-fingerprinting settings under `about:preferences#privacy`.

**Reply by vickygamer997** [reply, marked as answer]
It does not appear that there is a way to toggle the APIs, forced light mode, user agent, timezone changes, ect. separately.
